platform/linux-generic/odp_impl.c

```diff
@@ -5,17 +5,9 @@
 #include <odp/autoheader_internal.h>
 #include <odp/api/version.h>
 
-#define ODP_VERSION_IMPL 0
 #define ODP_VERSION_IMPL_STR \
-	_ODP_IMPLEMENTATION_NAME " " \
-	ODP_VERSION_TO_STR(ODP_VERSION_API_GENERATION) "." \
-	ODP_VERSION_TO_STR(ODP_VERSION_API_MAJOR) "." \
-	ODP_VERSION_TO_STR(ODP_VERSION_API_MINOR) "-" \
-	ODP_VERSION_TO_STR(ODP_VERSION_IMPL) " (v" \
-	ODP_VERSION_TO_STR(ODP_VERSION_API_GENERATION) "." \
-	ODP_VERSION_TO_STR(ODP_VERSION_API_MAJOR) "." \
-	ODP_VERSION_TO_STR(ODP_VERSION_API_MINOR) ") " \
-	ODP_VERSION_TO_STR(ODP_VERSION_BUILD)
+	_ODP_IMPLEMENTATION_NAME "-" \
+	ODP_VERSION_TO_STR(ODP_VERSION_BUILD) _ODP_GIT_REVISION \
```

Comment:
Extra backslash at the end. Also, the implementation version string could be formed directly in `odp_version_impl_str()` without adding this `ODP_VERSION_IMPL_STR` define.